portal/server/nunjucks-configuration/filter-localiseTimestamp.test.ts

```diff
@@ -1,6 +1,8 @@
-import { LONDON_TIMEZONE } from '../constants/date';
+import { IANA_TIMEZONES } from '@ukef/dtfs2-common';
```

Comment:
Some are importing `TIMEZONES`, others `IANA_TIMEZONES`, can they be consistent?